ChangeLog

```diff
@@ -5,6 +5,8 @@ mbed TLS ChangeLog (Sorted per branch, date)
 Bugfix
    * Fixes an issue with MBEDTLS_CHACHAPOLY_C which would not compile if
      MBEDTLS_ARC4_C and MBEDTLS_CIPHER_NULL_CIPHER weren't also defined. #1890
+   * Fix the internal definition of SSL_SOME_MODES_USE_MAC which previously
+     did not consider the presence of 3DES.
```

Comment:
Please describe the issue in a way that's meaningful to a user of the library (as opposed to us devs). Unless I'm mistaken, the user-visible effect of the bug was that CBC ciphersuites were not usable if DES was the only block cipher (usable with TLS) enabled at compile-time (the handshake returned "internal error").

The bug description should focus on user-visible behaviour, and if we want to mention the internal details of what went wrong, it should only come second (though in this particular case I don't think it's useful).